gulp/config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
         paths.modules + '/cf-pagination/src',
         paths.modules + '/cf-tables/src',
         paths.modules + '/cf-typography/src',
-        paths.modules + '/cfpb-chart-builder/src/static/css',
+        paths.modules + '/cfpb-chart-builder/src/**',
```

Comment:
Late to the party, but given the format of the paths above, do you need the `/**` here?